Sources/MlemMiddleware/FeedLoaders/User Content Feed Loader/PersonContentFeedLoader.swift

```diff
@@ -13,26 +13,25 @@ import Nuke
 /// just posts or just comments, and thus the standard Parent/Child FeedLoader construction does not work without
 /// severe API waste. This solution is a simplified variant of that architecture.
 ///
-/// The UserContentFeedLoader is the parent loader. It is responsible for all data fetching, and keeps track of two
-/// UserContentStreams, one for Posts and one for Comments. To load a page of items, it consumes and merges the child streams, just as
+/// The PersonContentFeedLoader is the parent loader. It is responsible for all data fetching, and keeps track of two
+/// PersonContentStreams, one for Posts and one for Comments. To load a page of items, it consumes and merges the child streams, just as
 /// in the standard Parent/Child FeedLoader; if either stream reaches the end of its items, it triggers a new load, the response from
 /// which is then incorporated into both child streams.
 
 @Observable
-public class UserContentFeedLoader: FeedLoading {
+public class PersonContentFeedLoader: FeedLoading {
```

Comment:
WRT [this comment](https://github.com/mlemgroup/MlemMiddleware/pull/34#discussion_r1690536416), we'd need a way to supply the first page of results to `PersonContentFeedLoader` from Mlem. This could be done in the initialiser or as an argument of `refresh` of something like that?

Reply:
Nice 👍